client/src/MongodbFunctions.js

```diff
@@ -0,0 +1,16 @@
+export const postNewItem = (itemName, itemId, weight, outByDate) => {
+    fetch('http://localhost:9000/items/addItem', {
```

Comment:
This URL will eventually need to be updated, right? I'm not sure what the best way to handle this is, though. Perhaps a variable somewhere which can point to localhost for now, and be changed to point to our deployment later?

Reply:
I have all the HTTP requests related to MongoDB in the MongodbFunctions.js file, so all the changes related to the endpoint URL will be in one centralized place.